rust/schema/annotations.py

```diff
@@ -119,7 +119,7 @@ class _:
     """
 
 
-@annotate(IfExpr)
+@annotate(IfExpr, cfg = True)
```

Comment:
It would be great if subtypes would "inherit" the `cfg` property from their base class. That way you don't have to repeat it for every class (with a risk of forgetting some).

Reply:
As long as one could override the annotation, that could work. For example, we don't want `ParensExpr` in the CFG.